functions-and-scope.js

```diff
@@ -14,6 +17,49 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan bijhouden?
 // Log het antwoord in de terminal.
 
+// Met een arrow function en return value
+const cumLaude = () => {
+    let cumLaude = null
```

Comment:
Als dit uiteindelijk een getal gaat worden, dan kun je ook beter met `0` beginnen
```suggestion
    let cumLaude = 0;
```